xtremedata-dbx-cluster-centos/azuredeploy.json

```diff
@@ -0,0 +1,452 @@
+            }
+        },
+        "domainName": {
+            "type": "String",
+            "defaultValue": "dbx",
+            "metadata": {
+                "description": "Prefix for dbX cluster public domain name"
+            }
+        },
+        "vnetName": {
+            "type": "String",
+            "defaultValue": "dbxvn",
+            "metadata": {
+                "description": "Prefix for dbX cluster network resources"
+            }
+        },
+        "adminPassword": {
+            "type": "SecureString",
+            "metadata": {
+                "description": "Password to login into dbX cluster nodes"
+            }
+        },
+        "vmSize": {
+            "type": "String",
+            "defaultValue": "Standard_DS14",
+            "allowedValues": [
+                "Standard_DS12",
+                "Standard_DS13",
+                "Standard_DS14"
+            ],
+            "metadata": {
+                "description": "dbX cluster node type"
+            }
+        },
+        "dataDiskSize": {
+            "type": "int",
+            "defaultValue": 128,
+            "allowedValues": [
+                128
+            ],
+            "metadata": {
+                "description": "Size of each of the two data disks, in GB"
+            }
+        },
+        "vmCount": {
+            "type": "int",
+            "defaultValue": 2,
+            "allowedValues": [
+                2, 4, 8
+            ],
+            "metadata": {
+                "description": "Number of nodes for dbX cluster"
+            }
+        }
+    },
+    "variables": {
+        "suffix": "x16",
+        "srcImage": "https://dbxpub.blob.core.windows.net:443/imgs/dbx-v4b-2-os-2015-07-15.vhd?se=2015-10-15T00%3A00%3A00Z&sp=r&sv=2014-02-14&sr=b&sig=iNDInvOf7Qab1iqAhISgadD3TEKGgtZl9kR9ZsHXDlQ%3D",
+        "vnetAddressSpace": "10.0.0.0/16",
+        "subnetAddressSpace": "10.0.0.0/24",
+        "subnetAddressPrefix": "10.0.0",
+        "subnetName": "Subnet-1",
+        "availabilitySetSettings": {
+            "name": "availabilitySet1",
+            "faultDomainCount": "3",
+            "updateDomainCount": "5"
+        },
+        "imageReferenceCentOS": {
+            "publisher": "OpenLogic",
+            "offer": "CentOS",
+            "sku": "7.1",
+            "version": "latest"
+        },
+        "storageAccountName": "[concat(parameters('newStorageAccountName'), variables('suffix') )]",
+        "osImage": "[concat('http://',parameters('newStorageAccountName'),variables('suffix'),'.blob.core.windows.net/img/os-disk-img.vhd')]",
+        "vnetName": "[concat(parameters('vnetName'), variables('suffix') )]",
+        "hostName": "[concat(parameters('hostName'), variables('suffix') )]",
+        "templateURLBase": "https://raw.githubusercontent.com/xtremedata/azure-quickstart-templates/master/xtremedata-dbx-cluster-centos",
+        "adminUsername": "azure-user",
+        "customData": "aGVsbG8gZnJvbSBPbGVnCg=="
```

Comment:
Hello to you too Oleg :) Is there any significance for this value? or is it just an easter egg? (base64 decodes to "hello from Oleg")